services/npm/npm-collaborators.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const { colorScale } = require('../../lib/color-formatters')
+const NpmBase = require('./npm-base')
+
+const keywords = ['node']
+
+const collaboratorColor = colorScale([2, 3, 4, 6])
+
+module.exports = class NpmCollaborators extends NpmBase {
```

Comment:
Same comment as above in `base` service about consistency between `maintainers` + `collaborators`